### ISSUE TITLE
Allow wallet file outside datadir?

### ISSUE THREAD

**Issue by Lucif666** [zcash/zcash#1873]
It would be great if client allowed wallet file outside data directory. Some of us prefer keeping wallets on encrypted flashdrives, which are usually small, while blockchain on large unencrypted harddrives.

**Comment by bitcartel** [comment]
@Lucif666 Does a symlink work?

**Comment by nathan-at-least** [comment]
Let's implement this as an experimental feature (so this depends on #2035).

**Comment by nathan-at-least** [comment]
@Lucif666 could you help us understand how to design this feature?

Do you need to migrate the same wallet between nodes?

Why do you want encryption? Is it to protect against theft *only* or also privacy? (e.g. if the node's machine is hacked while the wallet is offline, is it okay if the attacker can figure out your transaction history, addresses, etc…?)

**Comment by Lucif666** [comment]
Well, I just think it would be nice to have a possibility to have separate paths for wallet file and datadir. Like e.g. in monero gui - you can configure "wallet-path" and "data-dir" separately (or in Ethereum --keys-path and --datadir are separate). This way, you can take your wallet file away for example on a flash drive, which adds security and portability. Thank you.
